README.md

```diff
@@ -1,6 +1,81 @@
+# met-public
+
 [![Lifecycle:Experimental](https://img.shields.io/badge/Lifecycle-Experimental-339999)](https://github.com/bcgov/repomountie/blob/master/doc/lifecycle-badges.md)
 
 [![codecov](https://codecov.io/gh/bcgov/met-public/branch/main/graph/badge.svg?token=E1JXYU701O)](https://codecov.io/gh/bcgov/met-public)
 
-# met-public
-Modern Engagement Tool ops
+## The Modern Engagement Tools (MET) 
+
+This project is to develop modern engagement tools that address shortfalls identified in the Province’s existing public engagement processes.
+
+## Objectives 
+**The goal of this project is** to transform B.C.’s public engagement processes and enable the public to provide more value and influence decision-making while generating useful data insights, an enhanced post commenting experience for the user, and minimize rejected comments. The product will be cost effective, leverage investments and successes to date, explore best in class strategies, and be maintainable and continuously improved by government staff. It will be privacy enhancing, and support DRIPA/ GBA+ implementation and analysis, and continuously delivered and tested with users.
+
```

Comment:
Ran above through grammarly, got a few corrections if you'd like to consider:

**The goal of this project is** to transform B.C.’s public engagement processes and enable the public to provide more value and influence decision-making while generating useful data insights, an enhanced post-commenting experience for the user, and minimizing rejected comments. The product will be cost-effective, leverage investments and successes to date, explore best-in-class strategies, and be maintainable and continuously improved by government staff. It will be privacy-enhancing and support DRIPA/ GBA+ implementation and analysis, and continuously delivered and tested with users.

Reply:
![image](https://user-images.githubusercontent.com/91914654/187516232-1582fcae-ad0c-4653-a64b-832a1119f082.png)